src/main/resources/messages_en.properties

```diff
@@ -167,6 +167,7 @@ export.study.description.column.study.details=STUDY DETAILS
 export.study.description.column.ontology.id=ONTOLOGY ID
 export.study.description.column.dataset=DATASET
 cannot.exportAsXLS.lot.template=Can not export template as XLS. 
+cannot.exportAsXLS.germplasm.template=Can not export template as XLS.
```

Comment:
we use each word separated by .
I did not notice the one about lot does not follow the standar when merging that one